src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (days * 40 - 50);
+  } else if (days >= 3) {
+    return (days * 40 - 20);
+  } else {
+    return (days * 40);
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return (days * 40 - 50);
  }
  
  if (days >= 3) {
    return (days * 40 - 20);
  }
  
  return (days * 40);
```
числа(3, 7, 20, 40, 50) лучше вынести в переменные